src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -73,6 +75,15 @@ export class ProjectTokenHistoricalPriceService {
 
     const returnArray = new Array<IHistoricalPriceRecord>();
 
+    const startFundingTokenAmount = parseFloat(fromWei(lbpMgr.startingFundingTokenAmount, lbpMgr.fundingTokenInfo.decimals));
+    const startProjectTokenAmount = parseFloat(fromWei(lbpMgr.startingProjectTokenAmount, lbpMgr.projectTokenInfo.decimals));
```

Comment:
`startingProjectTokenAmount` includes fees.  I suspect that is a problem here. I can change it, let me know.